k8s.io/configmap-www-get.yaml

```diff
@@ -4,10 +4,11 @@ metadata:
   name: www-get
 data:
   # Adding new entries here will make them appear as files in the deployment.
+  # TODO: is there a better way to keep this up-to-date with cluster/get-kube.sh in kubernetes/kubernetes?
```

Comment:
that'd probably work, though we'd need to update our documentation (such as http://kubernetes.io/docs/getting-started-guides/binary_release/#download-kubernetes-and-automatically-set-up-a-default-cluster) to pass `-L` to `curl`, since by default it'll helpfully just print out "302 Found".